### PR TITLE
feat: Add Navbar, Sidebar, Toast

### PR DESCRIPTION
**Pull request by vigneshd332**
### Changes
<ul>
<li> Add Navbar, sidebar and toast notifs</li>
</ul>

### CODE REVIEW

File: src/components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBell } from '@fortawesome/free-solid-svg-icons';
+import './NavBar.css';
```
Comment (PriyanshuDangi) [inline src/components/NavBar/NavBar.tsx:3]: use css modules

File: src/components/SideBar/SideBar.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faCode,
+  faTrophy,
+  faGlobeAsia,
+  faCodeBranch,
+  faTv,
+  faSignOutAlt,
+} from '@fortawesome/free-solid-svg-icons';
+import './SideBar.css';
```
Comment (PriyanshuDangi) [inline src/components/SideBar/SideBar.tsx:10]: everywhere

File: src/components/NavBar/NavBar.tsx (hunk 1 above)
Comment (vigneshd332) [reply src/components/NavBar/NavBar.tsx]: done na, updated

**Comment by PriyanshuDangi** [comment]
image

**Review by CaptainIRS — CHANGES_REQUESTED** [review]
Add tooltips for the sidebar icons. You can use the [title](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title) attribute to keep things simple.

**Review by CaptainIRS — APPROVED** [review]
LGTM

**Pull request merged**